swagger.yaml

```diff
@@ -2416,6 +2416,10 @@ definitions:
         description: Json formatted string containing the user overrides for the install-config.yaml file
         example: '{"networking":{"networkType": "OVN-Kubernetes"},"fips":true}'
         x-go-custom-tag: gorm:"type:varchar(2048)"
+      controller_logs_collected_at:
+        type: string
+        format: datetime
```

Comment:
i think format is date-time ? we have the same issue with logs_update_at. I think it doesn't influence at all but for consistency in swagger let's change

Reply:
Changing it.